### PR TITLE
New version: Joplin.Joplin version 2.12.17

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/119808)

### CODE REVIEW

**Comment by brhahlen** [comment]
Hi @vedantmgoyal2009 
Thank you for all the work you do on getting Joplin in WinGet.
Could you check this error please?
I'm holding out on updating, because I want to do it via WinGet :-)

Thanks!

**Comment by ocyj** [comment]
@vedantmgoyal2009 thanks a lot for keeping winget packages up to date. It looks like the last two manifest versions got `x86` in all Installers > Architecture fields so `winget validate` fails with "Manifest Error: Duplicate installer entry found."

File: manifests/j/Joplin/Joplin/2.12.17/Joplin.Joplin.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+UpgradeBehavior: install
+ReleaseDate: 2023-09-14
+Installers:
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/laurent22/joplin/releases/download/v2.12.17/Joplin-Setup-2.12.17.exe
+  InstallerSha256: 02EECC0B3F00E50AAEF6427F3A605FF3AEAF9FB986CCEA1A594C865F32ED02AA
+  InstallerSwitches:
+    Custom: /CURRENTUSER /RESTARTAPPLICATIONS
+- Architecture: x86
```
Comment (kenden) [inline manifests/j/Joplin/Joplin/2.12.17/Joplin.Joplin.installer.yaml:22]: Change to:
`Architecture: x64`

File: manifests/j/Joplin/Joplin/2.12.17/Joplin.Joplin.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  Scope: machine
+  InstallerUrl: https://github.com/laurent22/joplin/releases/download/v2.12.17/Joplin-Setup-2.12.17.exe
+  InstallerSha256: 02EECC0B3F00E50AAEF6427F3A605FF3AEAF9FB986CCEA1A594C865F32ED02AA
+  InstallerSwitches:
+    Custom: /ALLUSERS /RESTARTAPPLICATIONS
+- Architecture: x86
```
Comment (kenden) [inline manifests/j/Joplin/Joplin/2.12.17/Joplin.Joplin.installer.yaml:34]: Change to:
`Architecture: x64`

**Review by kenden — COMMENTED** [review]
You have
- 2 entries (duplicate) with `Architecture: x86` and  `Scope: user`
- 2 entries (duplicate) with `Architecture: x86` and  `Scope: machine`

And no entries for `Architecture: x64`.

**Pull request closed without merging**